src/Console/Command/HelpCommand.php

```diff
@@ -122,8 +125,11 @@ protected function asText(ConsoleIo $io, iterable $commands): void
         foreach ($grouped as $prefix => $names) {
             $io->out("<info>{$prefix}</info>:");
             sort($names);
-            foreach ($names as $name) {
-                $io->out(' - ' . $name);
+            foreach ($names as $data) {
+                $io->out(' - ' . $data['name']);
+                if ($data['description']) {
+                    $io->info(str_pad(' \'', 5, '-') . ' ' . $data['description']);
```

Comment:
Should we use unicode here instead of the `'---` ascii art? We could use 
```suggestion
                    $io->info(str_pad(' \u{2514}', 15, '\u{2500}') . ' ' . $data['description']);
```
The additional length might need to be tweaked as each character is now 3 bytes.